src/pages/blog/starlingx-subcloud-local-install.md

```diff
@@ -0,0 +1,65 @@
+
+## What if?
+What if the artifacts could be made available on the edge cloud already, when the System Controller begins deployment? In other words, can we avoid downloading artifacts at the time of deployment?
+The answer is yes.
+Deployment over the edge clouds can be enhanced, if the artifacts for installation were already present on the physical machines that make up the edge clouds, prior to the deployment operations, in persistent storage. This, in essence, is local install.
+Local Install consists of three scenarios. The first is where a physical machine comes “prestaged” with installation artifacts right from the factory; the second is when the user wants to upgrade an edge cloud, and the third, when the user wants to reinstall.
+
+### Factory Installation:
+The artifacts for installation are copied to a persistent storage location on the machine at the factory. When adding an edge cloud, the system utilizes the prestaged artifacts instead of downloading them.
+The artifacts can also be used by an in-field technician in the form of an ISO image to enable prestaging on the edge cloud. 
+Upgrade
+In this case, the edge cloud is already up and running with a release version of StarlingX that is older than the one on the System Controller. A new version of StarlingX must be installed on the edge cloud, essentially an upgrade.
+Reinstall
+This is the case when both the System Controller and the edge cloud have the same version of StarlingX on them, and the user wants to re-install the edge cloud. At the end of the procedure, the edge cloud will have the same version as prior to the operation.
+
+In the case of Factory install, the installer identifies the prestaged artifacts at the prestage location and avoids downloading the artifacts when the edge cloud installation procedures are invoked. This is instrumental in managing network bandwidth use, as data transfers happen only when required. Since data transfer requirements still exist for upgrades/reinstall, this will reduce the installation time during the maintenance window, potentially reducing server downtime.
+In the remaining two cases, the artifacts can be transferred to the edge cloud servers through an out-of-band mechanism prior to scheduled maintenance times to prestage the system for install. Additionally, only new software versions can be transferred to reduce the amount of data transfer required. To avoid disrupting current network traffic, the data transfer can limit the bandwidth utilization during the transfer to the live system. 
+
+. Once the artifacts are prestaged in the appropriate location, upgrade and reinstall API can be invoked to complete the procedure while availing of the benefits of prestage.
+With this enhancement, the efficiency of installing, upgrading or re-installing is vastly improved.
+
+## Credits
```

Comment:
Maybe we could rather say 'Further contributors'?

Reply:
The folks under Credits internally reviewed and commented on the post; that's why I put it under Credits. I can change, if required, but I thought this was more appropriate.

Reply:
That's ok, we can leave it as is. Thank you for the additional information.